tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -633,11 +634,6 @@ tensorflow::Status CreateTRTNode(const std::vector<EngineInfo>& infos, int pos,
     return status;
   }
   // Add control input and input edges to the engine node.
```

Comment:
Please fix the comment. Also would you explain why removing the logic below? What problem will it cause?